spec/factories/activity.rb

```diff
@@ -101,6 +108,13 @@
 
       association :extending_organisation, factory: :delivery_partner_organisation
       association :reporting_organisation, factory: :beis_organisation
+
+      trait :with_report do
```

Comment:
Couldn't you move this outside of the `factory ..._activity do` block and have it top-level?

Reply:
Done, thanks for the pointer!